src/cpp/session/resources/dependencies/r-packages.json

```diff
@@ -385,20 +441,27 @@
         "shiny": {
             "description": "Shiny",
             "packages": [
-                "Rcpp",
```

Comment:
You've removed `Rcpp` completely rather than putting it in alphabetical order, was that intentional?

Reply:
Yes, Shiny no longer has a direct dependency on Rcpp.

https://github.com/rstudio/shiny/blob/2ce18ef3248bdeebdfadf7846fa61bfbf1b064a0/DESCRIPTION#L74-L97